1.illumination_correction/config_files/CHP-212_config.yml

```diff
@@ -0,0 +1,18 @@
+channels: # Keep empty since we are using a Harmony v7 output
```

Comment:
it might be also good to link to your GitHub issue for the inquisitive reader

Reply:
Great point! I will add!